libjsonutils/CMakeLists.txt

```
@@ -1,19 +1,24 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.13..3.16 FATAL_ERROR)
 project(libjsonutils VERSION 1.0.0 LANGUAGES CXX)
 
 #Make sure that custom modules like FindRapidJSON are found
 list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_SOURCE_DIR}/cmake)
 
 ##############################################
 # Declare dependencies
-find_package(Boost 1.55 REQUIRED COMPONENTS regex)
-find_package(RapidJSON 1.0 REQUIRED MODULE)
+set(MIN_BOOST_VERSION 1.71)
+find_package(boost_headers ${MIN_BOOST_VERSION} REQUIRED CONFIG)
+find_package(boost_regex ${MIN_BOOST_VERSION} REQUIRED CONFIG)
```

what about this?
```suggestion
find_package(Boost ${MIN_BOOST_VERSION} REQUIRED CONFIG COMPONENTS headers regex)
```
I would like to still have an example of finding a single components in a modularized library via `COMPONENTS`